src/main/java/aeneas/models/PuzzleLevel.java

```diff
@@ -11,10 +11,12 @@
   
   public PuzzleLevel(Bullpen bullpen) {
     super(bullpen);
+    board = new PuzzleBoard();
```

Comment:
Passing in a board allows us to test how a level will react to a specific board. It's the basic concept behind DI (dependency injection). You'll thank me when we start writing tests.